arcgis-ios-sdk-samples/Display information/Add graphics with renderer/AddGraphicsWithRendererViewController.swift

```diff
@@ -0,0 +1,171 @@
+    func addGraphicsOverlays() {
+        // Create a graphics overlay for the points.
+        let pointGraphicsOverlay = makePointGraphicsOverlay()
+        // Create a point graphic with `AGSPoint` geometry.
+        let pointGeometry = AGSPoint(x: 40e5, y: 40e5, spatialReference: .webMercator())
+        let pointGraphic = AGSGraphic(geometry: pointGeometry, symbol: nil)
+        // Add the graphic to the overlay.
+        pointGraphicsOverlay.graphics.add(pointGraphic)
+        
+        // Create a graphics overlay for the polylines.
+        let lineGraphicsOverlay = makeLineGraphicsOverlay()
+        // Create a line graphic with `AGSPolyline` geometry.
+        let lineBuilder = AGSPolylineBuilder(spatialReference: .webMercator())
+        lineBuilder.addPointWith(x: -10e5, y: 40e5)
+        lineBuilder.addPointWith(x: 20e5, y: 50e5)
+        let lineGraphic = AGSGraphic(geometry: lineBuilder.toGeometry(), symbol: nil)
+        // Add the graphic to the overlay.
+        lineGraphicsOverlay.graphics.add(lineGraphic)
+        
+        // Create a graphics overlay for the polygons.
+        let polygonGraphicsOverlay = makePolygonGraphicsOverlay()
+        // Create a polygon graphic with `AGSPolygon` geometry.
+        let polygonBuilder = AGSPolygonBuilder(spatialReference: .webMercator())
+        polygonBuilder.addPointWith(x: -20e5, y: 20e5)
+        polygonBuilder.addPointWith(x: 20e5, y: 20e5)
+        polygonBuilder.addPointWith(x: 20e5, y: -20e5)
+        polygonBuilder.addPointWith(x: -20e5, y: -20e5)
+        let polygonGraphic = AGSGraphic(geometry: polygonBuilder.toGeometry(), symbol: nil)
+        // Add the graphic to the overlay.
+        polygonGraphicsOverlay.graphics.add(polygonGraphic)
+        
+        // Create a graphics overlay for the curve shapes.
+        let curveGraphicsOverlay = makeCurveGraphicsOverlay()
+        let origin = AGSPoint(x: 40e5, y: 5e5, spatialReference: .webMercator())
+        // Create a heart-shape graphic from `AGSSegment`s.
+        let heartGeometry = makeHeartGeometry(center: origin, sideLength: 10e5)
+        let heartGraphic = AGSGraphic(geometry: heartGeometry, symbol: nil)
+        curveGraphicsOverlay.graphics.add(heartGraphic)
+        
+        // Add the overlays to the map view.
+        mapView.graphicsOverlays.addObjects(from: [pointGraphicsOverlay, lineGraphicsOverlay, polygonGraphicsOverlay, curveGraphicsOverlay])
+    }
+    
+    func makePointGraphicsOverlay() -> AGSGraphicsOverlay {
+        // Create a simple marker symbol.
+        let pointSymbol = AGSSimpleMarkerSymbol(style: .diamond, color: .green, size: 10)
+        // Create a graphics overlay for the points.
+        let pointGraphicsOverlay = AGSGraphicsOverlay()
+        // Create and assign a simple renderer to the graphics overlay.
+        pointGraphicsOverlay.renderer = AGSSimpleRenderer(symbol: pointSymbol)
+        return pointGraphicsOverlay
+    }
```

Comment:
I think the `make___GraphicsOverlay` functions could be generalized so you can pass the symbol type and use a switch statement. What do you think?

Reply:
Good point. I mostly copy pasted the code from the original implementation. Due to renaming of the file they don't show as diff but as deletions, so there is no way to compare them on GitHub.

I think this sample is intended to face those more beginner-level developers. So I intentionally didn't make big structural changes to the code, so that they can be copied and pasted easily. E.g. when someone want to test a single symbol, they can just copy the 2 methods (`makeSomeGraphicsOverlay` and `addGraphicsOverlays`) into a new file and see them renders.

I did extract them into `makeSomeGraphicsOverlay` methods though because otherwise the original `addGraphicsOverlay` method would be too long after adding the new curvature graphics. 😅 

I'll leave this discussion open to Phil and Sarat.